src/server/acl/acl_family.cc

```diff
@@ -152,25 +152,27 @@ void AclFamily::EvictOpenConnectionsOnAllProactorsWithRegistry(
   }
 }
 
+// Currently, Dragonfly supports only deleting one ACL user at a time.
```

Comment:
I did not notice that you can delete multiple users. I will take care of this :)

Reply:
Thanks a lot!